.eslintrc.json

```diff
@@ -36,6 +36,7 @@
       "rules": {
         "@typescript-eslint/no-use-before-define": 0,
         "class-methods-use-this": 0,
+        "no-useless-constructor": 0,
```

Comment:
this is required as eslint doesn't cope with kotlin style assigning constructors